views/survey.pug

```diff
@@ -28,8 +28,10 @@ html
                   if item.response
                     if item.response == 'text'
                         input(type='text' name=item.name placeholder='Type your answer' required= required)
-                    else if item.response == 'numeric'
+                    else if item.response == 'number'
                       input(type='number' name=item.name placeholder='Enter a number' required= required)
+                    else if item.response == 'date'
+                      input(type='date' name=item.name placeholder='Enter the date (yyyy-mm-dd)' required= required)
```

Comment:
Wording suggestion:
```suggestion
                      input(type='date' name=item.name placeholder='Enter a date' required= required)
```

I think in most modern browsers it will make a date picker. I'm not sure what happens in older browsers. Would you mind reading up on that just to see that we don't need any extra scripts or formatting guides for it?

Reply:
I think we have to manually code a date picker. I included the formatting because of this discussion: https://stackoverflow.com/questions/55839778/why-isnt-date-showing-in-pug-form-input.

Reply:
OK I think they're solving a different problem — getting a variable to display as a date in a date picker.

Can you show me what it looks like on your end when you use this? When I have tested date picker rendering it seems to automatically work in most modern browsers.

Reply:
I've not been able to test the code for the last few days due to an issue with an update. I'll fix it by tomorrow and get back to you.

Reply:
Ok. Post the error in slack if you get stuck.

Reply:
Yep you're right, the date picker comes up, no need for the excess wording.